daemon/container_operations.go

```diff
@@ -1008,6 +1007,9 @@ func (daemon *Daemon) releaseNetwork(container *container.Container) {
 		return
 	}
 
+	container.NetworkSettings.SandboxID = ""
+	container.NetworkSettings.SandboxKey = ""
```

Comment:
I see we don't reset `container.NetworkSettings.SandboxKey` if we find that `container.NetworkSettings.SandboxID` is empty. Could there be any situations where it's non-empty in that case (and would _not_ resetting it cause issues in that situation?)

Reply:
(considering if we should move the resetting above the `if sid == ""` branch above if there could be such situations, and if those would be problematic.

Reply:
The only other place where `SandboxID` is written is here:

https://github.com/moby/moby/blob/076c976e28cac7802835e944ae03cee2662cbea2/daemon/container_operations.go#L1160-L1164

Before this commit, `SandboxID` was never cleared. So, the purpose of this condition is to prevent calling `releaseNetwork` twice or more on the same container. I think it's safe to not check them / clear them separately.